### PR TITLE
Add contribution guide, README, and CI

### PR DESCRIPTION
**Pull request by fghaas**
Add information about where the documentation is hosted, and how one can contribute to it.

Also, add a README for those who just stumble upon this repo, and include a GitHub Actions workflow that runs the `tox` tests, so we can enable branch protection.

### CODE REVIEW

File: .github/workflows/tox.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v1
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Install dependencies
+        run: |
+          pip install tox tox-gh-actions
```
Comment (evrardjp) [inline .github/workflows/tox.yml:14]: Probably should be --user, but who cares :)

File: .github/workflows/tox.yml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+      - name: Test with tox
+        # Explicitly specifying the testenvs shouldn't really be
+        # necessary here, but for some reason tox invokes no testenvs
+        # at all if invoked as just "tox" from the GitHub Actions
+        # workflow. Until that is fixed, name the testenvs.
+        run: tox -e gitlint,yamllint,build
```
Comment (evrardjp) [inline .github/workflows/tox.yml:20]: Should we gather the artifacts and display them somewhere, to help at reviewing?

**Review by evrardjp — COMMENTED** [review]
I think I would have added the CI into a different PR... But let's not bikeshed here.

File: .github/workflows/tox.yml (hunk 2 above)
Comment (fghaas) [reply .github/workflows/tox.yml:22]: For now I've only made them downloadable as `build-3.8.zip`, `build-3.9.zip`, and `build-3.10.zip`.

**Comment by fghaas** [comment]
> I think I would have added the CI into a different PR... But let's not bikeshed here.

No, I want the branch protection rules to apply straight away, that's why I want the Actions workflow in this one.

File: .github/workflows/tox.yml (hunk 1 above)
Comment (fghaas) [reply .github/workflows/tox.yml:16]: Exactly.

**Comment by fghaas** [comment]
OK, checks are implemented and pass, and they also include site builds as downloadable artifacts. Good to review and approve now.

**Comment by fghaas** [comment]
> OK, checks are implemented and pass, and they also include site builds as downloadable artifacts. Good to review and approve now.

@evrardjp I just realized that you assigned this to me, so I'll go ahead and self-approve if GitHub lets me do that.

**Comment by fghaas** [comment]
> I just realized that you assigned this to me, so I'll go ahead and self-approve if GitHub lets me do that.

It doesn't. @evrardjp, please approve.

**Pull request merged**